extensions/net/lib.rs

```diff
@@ -89,7 +89,18 @@ pub fn get_unstable_declaration() -> PathBuf {
   PathBuf::from(env!("CARGO_MANIFEST_DIR")).join("lib.deno_net.unstable.d.ts")
 }
 
-pub fn init<P: NetPermissions + 'static>(unstable: bool) -> Extension {
+/// `NoCertificateValidation` is a struct so it can be placed inside `GothamState`;
+/// using type alias for a `Option<Vec<String>>` could work, but there's a high chance
+/// that there might be another type alias pointing to a `Option<Vec<String>>`, which
+/// would override previously used alias.
```

Comment:
You can use wrapper struct instead `pub struct NoCertificateValidation(Option<Vec<String>>)`. This comment is valid for all types being put in `GothamState` and there are already several comments about it.